pkgs/by-name/pr/prismlauncher-unwrapped/package.nix

```diff
@@ -4,12 +4,11 @@
 , stripJavaArchivesHook
```

Comment:
I prefer this way:

1. A commit that makes the necessary changes so that the package becomes suitable to be migrated to by-name;
2. A commit that migrates the package to by-name.

The granularity is useful for bisection- and review-related things.

Reply:
i'll keep this in mind for the future